templates/vr-ar.jade

```diff
@@ -88,15 +89,111 @@ block main
             div.Grid-cell.Grid-cell--full
               p We also maintain and contribute to a community driven repository of tools for eye tracking in head mounted displays called #[a(href="https://gitter.im/pupil-labs/hmd-eyes" title="HMD-eyes on Github" rel="noopener") HMD-eyes]
 
-    //- community
-    div.Site-content-container
+
+    section.Site-content-container
+      a(name="technical-specs")
       div.Feature-container-column
         div.Grid.Grid--justifyCenter.Grid--center.Grid--alignContentCenter.Feature-row
-          div.Grid-cell.Grid-cell--full.u-textCenter
+          div.Grid-cell.Grid-cell--full
             div.Feature
-              h1.Feature-title--lg.u-padBottom--1 Community
-          div.Grid-cell.Grid-cell--full.u-textCenter
-            p Our goal is to work with and support a community of researchers and developers in creating reusable open source software for eye tracking in HMDs. 
-            p Many researchers and developers have already thought of, or implemented eye tracking software HMDs. Instead of continuing to reinvent the wheel, we have formed a community with the goal of sharing knowledge and tools.
-            p Fork the hmd-eyes code on  #[a(href="https://github.com/pupil-labs/hmd-eyes" title="hmd-eyes on Github" rel="noopener") Github]. Chat with the hmd-eyes community on #[a(href="https://gitter.im/pupil-labs/hmd-eyes" title="HMD-eyes Gitter chat" rel="noopener") Gitter].
+              h1 Technical Specifications & Performance
+              h4 There are subtle differences in hardware add-ons that share technical and performance specifications, or otherwise noted. 
```

Comment:
VR and AR add-ons share many of the same specifications due to our modular design approach. There are subtle differences in specs and performance where noted in the table below.